### ISSUE TITLE
Issue report on LibreHealth Updater Administration modal

### ISSUE THREAD

**Issue by ahanipeitra1994** [LibreHealthIO/lh-ehr#1564]
A)   **Outreachy**  Username : Peitra.
B)   **Issue Title**: The Updater Administration modal not  noticeable.
C)   **Report Date**:  03/13/2020.
D)  **Sites Affected:** Local  Instance,
E)   **OS/browser used**:  windows10/ Mozilla FireFox.
F)   **EHR Work Flow Module**: Updater  icon -> Updater  Administration 
G)  **Steps to Reproduce the Issue:**
1)	Hover over the  “ Updater Icon” bottom right of the Page.
2)	Click on “ Updater Setting” .
3)	Click on “ Updater Administration”
4)	Click on “ Add User or Remove User”

H)  **Expected Behavior**: The modal should appear on the  “Updater Administration Page” with a clear view.
I)  **What Actually Happened**:  The modal appears on the bottom right of the Page which does not portray the notification clearly.So a good view of the modal is difficult.
 J)  **Screen With Explanation.**

![image](https://user-images.githubusercontent.com/60817596/76679044-6b43ba80-65dd-11ea-8ceb-cdb5e36c4708.png)
  
K)  **Estimation of Severity**: Minor distraction or confusing but workflow can continue correctly.
L)  **Solution:** The modal should be placed at a visible place using Java Script.

**Comment by muarachmann** [comment]
I in particular ... not sure is standard rely on my notifications being shown on the **top-right, bottom-right.** . Don't know if this is any issue to others. Nevertheless a global for this should do

**Comment by robbyoconnor** [comment]
No. You cannot propose a project like this. Additionally this isn't the place to do it.

**Comment by robbyoconnor** [comment]
Did you discuss this before posting?
